### ISSUE TITLE
Sign in for more options in online-available journals

### ISSUE THREAD

**Issue by kmerguerian** [NEU-Libraries/scholar-onesearch#264]
In the modal, for journal articles that we have, the links are presented under a "View It" heading, followed by a yellow banner that says "Sign in for more options".  Can this be removed?  We don't have it in the current system, and there are no more options anyway. 

See 

http://northeastern-primo-sandbox.hosted.exlibrisgroup.com/primo-explore/fulldisplay?docid=TN_hindawi10.1155/2010/915937&context=PC&vid=NUdev&search_scope=new_everything_scope&tab=default_tab&lang=en_US

**Comment by winthropo** [comment]
In the back office I only see a way to relabel it not remove it.

**Comment by kmerguerian** [comment]
What if you just put a blank in the label?

**Comment by winthropo** [comment]
you can't do that. i'll keep looking into this. seems minor to me, but if this is something that needs fixing i'll keep looking at it.

**Comment by kmerguerian** [comment]
Why would it be different from the old UI?

Karen

**Comment by winthropo** [comment]
because it is the new UI??

i'm not sure what you are asking.

**Comment by kmerguerian** [comment]
Oh, I know!  How about changing the label to say, "You may need to sign in to view or request this material."  Then it would apply whether we have the material or not.

Karen

**Comment by winthropo** [comment]
i'm confident we can fix this.

**Comment by winthropo** [comment]
Because that is way too wordy in my opinion.

**Comment by kmerguerian** [comment]
It just seems weird ... I know there are grey areas, but to me,  functionality, algorithms, presentation of options are all features that are separate from design. I know there are grey areas, but if I chose a new theme for my gmail, that doesn't change, or require me to change, other settings related to functionality.  Those functional settings are separate, and they are developed and customized independently.

So I hope the leadership of the library appreciates the extent to which we are not just customizing the design and branding, but reviewing and making decisions about features and settings that go beyond design.

Not to be too meta, but it's also interesting how these github tickets turn into conversations about things that aren't necessarily related to the original issue, or even solvable at all in the github environment.

Karen

**Comment by kmerguerian** [comment]
Right, once you're down to talking about number of words you know there's a solution on the horizon!

**Comment by kmerguerian** [comment]
Summary of conversation in our group:

1. This is Alma CSS so Eli may not be able to do the same customizations, such as removing this label in certain situations. 

2. BU has called this "Request items and Save Results...sign in" however, it is very prominent and it is in most cases, where the item can be viewed online, it's not the place to request. 

I will cogitate on this and propose a label, something very similar to what BU has, by February 15 so it can be done by our March beta.

**Comment by kmerguerian** [comment]
Request and save options?

**Comment by kmerguerian** [comment]
Request and save...

**Comment by kmerguerian** [comment]
Is this helpful?

http://knowledge.exlibrisgroup.com/Alma/Knowledge_Articles/Control_when_ILLiad_request_appears_in_Primo's_GetIt_tab_-_guest_users%2C_item_availability

Alma Menu > Fulfillment > Fulfillment Configuration > Configuration Menu > Discovery Interface Display Logic > Display Logic Rules

**Comment by winthropo** [comment]
I don't see how it is relevant to this Git issue.

**Comment by winthropo** [comment]
@kmerguerian maybe you can explain this to me in person. I don't know where we are with this one.

**Comment by winthropo** [comment]
READ discussed this on June 8th and agreed we like the way this is working now. But I will leave the case open.
